builder/builder.go

```diff
@@ -1838,6 +1838,12 @@ func (b *Builder) AddRPMList(rpms []string) error {
 		if _, err := os.Stat(repoPath); err == nil {
 			continue
 		}
+		// Remove source RPMs because they should not be added to mixes
+		if strings.Contains(rpm, ".src.rpm") {
```

Comment:
Optional: This could be `strings.HasSuffix` to be a little faster.

Reply:
My selective reading thought this wasn't a suffix for some reason, thanks :) changing.